haproxy/state/from_ha.go

```diff
@@ -51,30 +51,44 @@ func FromHAProxy(ha HAProxyRead) (State, error) {
 		if err != nil {
 			return state, err
 		}
-		if len(logTargets) > 1 {
-			return state, fmt.Errorf("expected at most 1 filter for frontend %s, got %d", f.Name, len(filters))
+		if len(logTargets) >= 2 {
```

Comment:
No change (">1" == ">=2")? Also, why is logTargets reused here? Looks like there was a typo initially, we should check length of filters here.